Setup/UpgradeSchema.php

```diff
@@ -11,13 +14,22 @@
 use Razorpay\Subscription\Model\ResourceModel\Subscriptions;
 use Razorpay\Subscription\Model\ResourceModel\SubscriptionsOrderMapping;
 /**
- * @UpgradeSchema to update the current db with subscription 
+ * @UpgradeSchema 
+ * update the current db with subscription 
```

Comment:
Did you tested without this `Setup/UpgradeSchema.php`

Reply:
yes